src/validators/max-length.ts

```diff
@@ -0,0 +1,37 @@
+import { LengthArgs, parseLengthParams, isLengthValid } from './length';
+import { FieldValidationFunctionSync } from '../model';
+
+export const VALIDATOR_TYPE = 'MAX_LENGTH';
+
+let defaultMessage = 'The value provided does not fulfil max length';
+export const setErrorMessage = message => (defaultMessage = message);
+
+const BAD_PARAMETER =
+  'FieldValidationError: Parameter "length" for maxLength in customArgs is mandatory and should be a valid number. Example: { length: 4 }.';
+
+const DEFAULT_PARAMS: LengthArgs = null;
+
+function isStringLengthValid(value: string, length: number): boolean {
+  return value.length <= length;
+}
```

Comment:
Why not fat arrow anonymous function for isStringLengthValid?

Reply:
Ported from the other project, I think is a good idea to port it to fat arrow, going to udpated it., Done